src/docfx/build/toc/TableOfContentsLoader.cs

```diff
@@ -194,6 +220,8 @@ private void TraverseAndMerge(TableOfContentsNode node, List<SourceInfo<TableOfC
                 Items = items,
             };
 
+            newNode.LandingPageType = node.Value.LandingPageType;
```

Comment:
move this line to `TableOfContentsNode` constructor.

Reply:
Fixed.